### ISSUE TITLE
Support the `pcss` extension

### ISSUE THREAD

**Issue by ZeeCoder** [prettier/prettier#3629]
**Prettier 1.9.2**

**Input:**
```pcss
// a file with ".pcss" extension
.SomeClass {
    
}
```

**Output:**
```
[error] > 1 | .SomeClass {
[error]     | ^
[error]   2 |
[error]   3 | }
[error]   4 |
```

**Expected behavior:**
Should compile fine, as css / less / sass does.

**Comment by ikatyang** [comment]
It should be fixed by #3454, and will be included in the next release.

**Comment by ZeeCoder** [comment]
Awesome, thanks!
